appservice/src/main/java/nu/yona/server/analysis/rest/UserActivityController.java

```diff
@@ -540,19 +550,20 @@ public ActivityForOneUserResource toResource(ActivityForOneUser dayActivity)
 			ActivityForOneUserResource dayActivityResource = instantiateResource(dayActivity);
 
 			UUID goalId = dayActivity.getGoalId();
-			UUID userId = goalIdMapping.getUserId();
+			UUID requestingUserId = goalIdMapping.getUserId();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "requestingUserId" which hides the field declared at line 534. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AHiddenFieldCheck)